webapp/graphite/finders/standard.py

```diff
@@ -68,16 +68,24 @@ def _find_paths(self, current_dir, patterns):
     match the corresponding pattern in patterns"""
     pattern = patterns[0]
     patterns = patterns[1:]
-    try:
-      entries = os.listdir(current_dir)
-    except OSError as e:
-      log.exception(e)
-      entries = []
 
+    has_wildcard = pattern.find('{') > -1 or pattern.find('[') > -1 or pattern.find('*') > -1
```

Comment:
A glob technically also can have a `?`, not sure if it's possible to pass it through the render API though.